engine/src/history.rs

```diff
@@ -85,19 +86,16 @@ impl History {
                 println!("Couldn't open file because: {}", e);
             }
         }
-        history
+        Ok(history)
     }
 
     pub fn write_move(&self, file_name: &str, turn: usize, board_move: String) {
```

Comment:
you might consider moving the formatting elements of this into an [`impl Display`](https://doc.rust-lang.org/std/fmt/trait.Display.html)